pages/models.py

```diff
@@ -1306,6 +1306,7 @@ def add_group_view(self, request):
 
                 # Get new group id
                 group.group_id = str(self.get_new_group_id())
+                group.group_type = 'local-group'
```

Comment:
Why do we need this is we already have a default set in the model for this field?